core/src/main/java/io/questdb/cairo/mv/MatViewDefinition.java

```diff
@@ -48,13 +48,17 @@
 import static io.questdb.std.datetime.microtime.Timestamps.MINUTE_MICROS;
 
 public class MatViewDefinition implements Mutable {
-    // For now, incremental refresh is the only supported refresh type.
+    public static final int INCREMENTAL_INTERVAL_REFRESH_TYPE = 1;
```

Comment:
I'd call it a timer or periodic refresh rather than an interval refresh. Interval is confusing in this context, interval can be the timestamp range data interval or the refresh period interval.